app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-Here is a single user
\ No newline at end of file
+<div>
+  <div style='display:flex;'> 
+    <img src=<% @user.photo %> />
+    <div>
+    <h3><%= @user.name %></h3>
+    <p>Number of Posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+  <div>
+    <h4>Bio</h4>
+    <p><%= @user.bio %></p>
+  </div>
+  <ul>
+     <% @user_posts.each do |post| %>
+        <li> 
+           <h3><%= post.title %></h3>
+           <p><%= post.text %></p>
+           <p>Likes: <%= post.likes_counter %></p>
+           <p>Comments: <%= post.comments_counter %></p>
+        </li>
+      <% end %>
+  </ul>
+</div>
```

Comment:
- [ ] [REQUIRED] You have a nice implementation here as well @Trust-007 , however, you need to ensure that **See all Post** links opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png). It looks like you forgot to implement this part as well. Kindly implement this.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="317" alt="Screenshot 2023-03-01 at 23 11 57" src="https://user-images.githubusercontent.com/58692590/222281725-06b01735-9ea0-4cc8-b663-89b8e0e8594d.png"> | <img width="414" alt="Screenshot 2023-03-01 at 23 39 42" src="https://user-images.githubusercontent.com/58692590/222281865-0c682d81-bd97-4336-8764-ed16aec6c635.png">
- Remember to add the `See All Posts` button to this show page so that when he user clucks the button, it takes them to the user's [post index page](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).